### DISCUSSION TITLE
PRO - cancel membership problem

### DISCUSSION THREAD

**Discussion by Beowolve** [cssbattle/forum#320] [Bugs reports]
I have canceled PRO membership on 30.10.2022 and after that lost PRO status instantly. But I have purchased yearly on 8.11.2021 and would have expected keeping PRO the full year.
Anyway I did report that on Discord and today all of a sudden I have PRO status again which let me think that something got fixed/changed.

But if I go to "Membership" now, it still says next payment (yearly) occurs on Nov. 8. 2022.

![image](https://user-images.githubusercontent.com/32326640/199968831-812f106a-4f9c-4143-9034-5f643143d617.png)

So I tried to use cancel membership again. But now I get a page not found message on a "*.paddle.com" site.
![image](https://user-images.githubusercontent.com/32326640/199968940-f2364e67-0c8a-45de-8bf3-96d5d4c520dd.png)

This does not look right.

**Comment by chinchang** [comment]
Hi @Beowolve 
Sorry about the confusion here. After you reported on Discord, we started fixing this thing. As part of it, your PRO membership was revived, till 8th Nov 2022 of course.
The UI as you saw isn't still appropriate...something which we are working on fixing right now. Rest assured, your subscription is cancelled and you won't be charged

**Reply by Beowolve** [reply]
Thx

**Reply by JohnSmith-LT** [reply]
Hi @chinchang 

Oops, I assumed that this was fixed from the messages above and like @Beowolve I cancelled my Pro membership expecting to keep it until October 2023 but it has now gone completely... can this please be reinstated? CSSBattle name is Smitcher...

**Reply by chinchang** [reply]
seems like some issue is still there 🤔  Fixed for you @JohnSmith-LT
